autoload/neomake/makers/ft/go.vim

```diff
@@ -1,7 +1,7 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#go#EnabledMakers() abort
-    return ['go', 'golint', 'govet']
+    return ['go', 'golint', 'govet', 'gometalinter']
```

Comment:
Then the first two commits of https://github.com/neomake/neomake/pull/1213 make sense..  let's discuss it there further.